atlasdb-dbkvs-hikari/src/main/java/com/palantir/nexus/db/pool/config/OracleConnectionConfig.java

```diff
@@ -112,6 +114,10 @@ public Properties getHikariProperties() {
 
         props.setProperty("oracle.jdbc.maxCachedBufferSize", "100000");
 
+        if (getDefaultRowPrefetch().isPresent()) {
+            props.setProperty("defaultRowPrefetch", getDefaultRowPrefetch().get().toString());
```

Comment:
I would also prefer an explicitly set default defaultRowPrefetch , if not just to force synergy between postgres / oracle